nestkernel/exceptions.cpp

```diff
@@ -408,3 +408,12 @@ nest::NumericalInstability::message() const
       << "updating " << model_ << ".";
   return msg.str();
 }
+
+std::string
+nest::NoEntryToMap::message() const
```

Comment:
The name of the exception seems misleading. Isn' t this a `KeyError`?

Reply:
Changed to KeyError and made the error message more generic in eb41277